test_httpbin.py

```diff
@@ -265,6 +265,29 @@ def test_set_cors_allow_headers(self):
         self.assertEqual(
             response.headers.get('Access-Control-Allow-Headers'), 'X-Test-Header'
         )
+
+    def test_headers(self):
+        headers = {
+            "Accept": "*/*",
+            "Host": "localhost:1234",
+            "User-Agent": "curl/7.54.0",
+            "Via": "bar"
+        }
+        response = self.app.get('/headers', headers=headers)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue({'Accept', 'Host', 'User-Agent'}.issubset(set(response.json.keys())))
```

Comment:
It may be good to have an explicit check here that `Via` is not included in this set. Otherwise, these two tests could return the same header set and we wouldn't know.

Reply:
@nateprewitt Added an explicit check on L279.